src/site/content/en/blog/reporting-api/index.md

```diff
@@ -0,0 +1,664 @@
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - monitoring
+  - security
+---
+
+{% Banner 'caution', 'body' %} This is an API guide with detailed usage examples for the
+**Reporting API (v1)**, which uses the `Reporting-Endpoints` header. If you're already
+using the legacy Reporting API with the `Report-To` header and need to migrate to the new
+version (v1), head over to the [migration post](/reporting-api-migration) instead. {%
+endBanner %}
+
+Some errors only occur in production. You never see them locally or during development
+because **real users**, **real networks**, and **real devices** change the game. The
+Reporting API helps catch some of these errors across your site and transmit them to an
+endpoint you've specified.
+
+This API enables you to activate a reporting mechanism that's **operated by the browser**.
+When the Reporting API is configured on your site, **reports can be generated** when certain
+policies are violated (or when deprecated APIs are used, when the browser intervenes on
+your site or crashes, and more). The browser then sends those report(s) to the
+**endpoint(s)**, that is report collectors, you've configured.
+
+Setting up the Reporting API gives you peace of mind that when users experience these
+types of unforeseen errors, you'll be aware and able to fix them if needed. ✨
+
+```http
+  Reporting-Endpoints: main-endpoint="https://reports.example/main", default="https://reports.example/default"
+  Content-Security-Policy: script-src 'self'; object-src 'none'; report-to main-endpoint;
+```
+
+This post covers what this API can do, how it works, and how you can use it.
+
+Let's dive in!
+
+## Demo and code
+
+💥 [Demo site: report generation page](https://reporting-api-demo.glitch.me). This page
+uses the new Reporting API via the `Reporting-Endpoints` header. See the
+[code](https://glitch.com/edit/#!/reporting-api-demo).
+
+📬 [Demo site: reporting endpoint](https://reports-endpoint.glitch.me). This server
+receives and displays reports. See the
+[code](https://glitch.com/edit/#!/reports-endpoint).
+
+## Use cases and report types
+
+{% Banner 'caution', 'body' %}
+
+[Network Error Logging](https://w3c.github.io/network-error-logging/) isn't listed because it isn't supported in the new version of
+the API. Check the [migration guide](/reporting-api-migration/#network-error-logging) for details.
+
+{% endBanner %}
+
+<div class="w-table-wrapper">
+  <table>
+    <thead>
+      <tr>
+        <th>Report type</th>
+        <th>Example of a situation where a report would be generated</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP">CSP</a> violation (Level 3 only)</td>
+        <td>You've set a <code>Content-Security-Policy</code> on your site, but one of your pages is trying to load a script that's not allowed by your CSP.</td>
```

Comment:
I think a CSP is set against a page rather than a site, so `one of your pages` is somewhat confusing statement. I'd rephrase it to `You've set a <code>Content-Security-Policy</code> on one of your pages, but the page is trying to load a script that's not allowed by your CSP.`